### PR TITLE
netskope legacy collection

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X ] Update Content - Revisions and updating sections
- [X ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/security-threat-detection/netskope-legacy-collection.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+Before creating the HTTP source, identify the Sumo Logic Hosted Collector you want to use, or create a new Hosted Collector as described in the following task.
+When you configure the HTTP Source, make sure to save the HTTP Source Address URL. You will need this to configure in configuration file.
+**To add a hosted collector and HTTP source, do the following:
+1. To create a new Sumo Logic Hosted Collector, perform the steps in [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+2. Add an [ HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics).
+3. In **Advanced Options for Logs**, under Timestamp Format, click **Specify a format** and enter the following:
+* Specify **Format** as `epoch`
+* Specify **Timestamp locator** as `\"timestamp\": (.*),`
+
+
```
Comment (stacykor) [inline docs/integrations/security-threat-detection/netskope-legacy-collection.md:58]:
```suggestion
```

File: docs/integrations/security-threat-detection/netskope-legacy-collection.md [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+| count by app_session_id
+| count
+```
+
+
+## Installing the Netskope App
+
+This section demonstrates how to install the Netskope App, and provides examples and descriptions for each of the dashboards. The Netskope App has the following components:
+
+* **Application Usage:** Insights into application usage; specifically by devices, users, users and traffic patterns.
```
Comment (stacykor) [inline docs/integrations/security-threat-detection/netskope-legacy-collection.md:298]:
```suggestion
* **Application Usage.** Insights into application usage; specifically by devices, users, users and traffic patterns.
```

File: docs/integrations/security-threat-detection/netskope-legacy-collection.md [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+* **Security Alerts:** Visibility into Netskope security alerts and violations and the ability to identify effects of a breach.
```
Comment (stacykor) [inline docs/integrations/security-threat-detection/netskope-legacy-collection.md:299]:
```suggestion
* **Security Alerts.** Visibility into Netskope security alerts and violations and the ability to identify effects of a breach.  
```

**Review by stacykor — CHANGES_REQUESTED** [review]
Looks like an extra line in the steps. Please check.

File: docs/integrations/security-threat-detection/netskope-legacy-collection.md [hunk 4]
```diff
@@ -0,0 +1,439 @@
...
+
+#### Prerequisite
+Before creating the HTTP source, identify the Sumo Logic Hosted Collector you want to use, or create a new Hosted Collector as described in the following task.
+When you configure the HTTP Source, make sure to save the HTTP Source Address URL. You will need this to configure in configuration file.
+**To add a hosted collector and HTTP source, do the following:
+1. To create a new Sumo Logic Hosted Collector, perform the steps in [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+2. Add an [ HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics).
+3. In **Advanced Options for Logs**, under Timestamp Format, click **Specify a format** and enter the following:
+* Specify **Format** as `epoch`
+* Specify **Timestamp locator** as `\"timestamp\": (.*),`
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/netskope-legacy-collection.md:56]:
```suggestion
   * Specify **Format** as `epoch`
   * Specify **Timestamp locator** as `\"timestamp\": (.*),`
```

File: docs/integrations/security-threat-detection/netskope-legacy-collection.md [hunk 5]
```diff
@@ -0,0 +1,439 @@
...
+
+This section demonstrates how to add a hosted Sumo Logic collector and HTTP Logs and Metrics source, to collect events for Netskope
+
+
+#### Prerequisite
+Before creating the HTTP source, identify the Sumo Logic Hosted Collector you want to use, or create a new Hosted Collector as described in the following task.
+When you configure the HTTP Source, make sure to save the HTTP Source Address URL. You will need this to configure in configuration file.
+**To add a hosted collector and HTTP source, do the following:
+1. To create a new Sumo Logic Hosted Collector, perform the steps in [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+2. Add an [ HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics).
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/netskope-legacy-collection.md:53]:
```suggestion
2. Add an [HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics).
```

File: docs/integrations/security-threat-detection/netskope-legacy-collection.md [hunk 6]
```diff
@@ -0,0 +1,439 @@
...
+
+### Step 1: Adding a Hosted Collector and HTTP Source (DEPRECATED)
+
+This section demonstrates how to add a hosted Sumo Logic collector and HTTP Logs and Metrics source, to collect events for Netskope
+
+
+#### Prerequisite
+Before creating the HTTP source, identify the Sumo Logic Hosted Collector you want to use, or create a new Hosted Collector as described in the following task.
+When you configure the HTTP Source, make sure to save the HTTP Source Address URL. You will need this to configure in configuration file.
+**To add a hosted collector and HTTP source, do the following:
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/netskope-legacy-collection.md:51]:
```suggestion
To add a hosted collector and HTTP source, do the following:
```

File: docs/integrations/security-threat-detection/netskope-legacy-collection.md [hunk 7]
```diff
@@ -0,0 +1,439 @@
...
+* Specify **Timestamp locator** as `\"timestamp\": (.*),`
+
+
+1. Click **Add**.
+
+
+### Step 2: Getting a token from the Netskope Portal (DEPRECATED)
+
+Netskope REST APIs use an auth token to make authorized calls to the API. This section demonstrates how to obtain a token from the Netskope user interface (UI).
+**To obtain a Netskope auth token, do the following:
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/netskope-legacy-collection.md:65]:
```suggestion
To obtain a Netskope auth token, do the following:
```

File: docs/integrations/security-threat-detection/netskope-legacy-collection.md [hunk 8]
```diff
@@ -0,0 +1,439 @@
...
+1. Login to Netskope as the Tenant Admin.
+2. Go to the API portion of the Netskope, **Settings > Tools > Rest API**.
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/netskope-legacy-collection.md:67]:
```suggestion
2. Go to the API portion of the Netskope, **Settings** > **Tools** > **Rest API**.
```

File: docs/integrations/security-threat-detection/netskope-legacy-collection.md [hunk 9]
```diff
@@ -0,0 +1,439 @@
...
+### Updating the Sumo Logic Netskope collector (DEPRECATED)
+
+Sumo Logic periodically makes changes to the collector. To make sure your collector is up-to-date, perform the following tasks on each machine running the collector:
+* Check the latest version of the Netskope collector on this page: [https://pypi.org/project/sumologic-netskope-collector/#history](https://pypi.org/project/sumologic-netskope-collector/#history)
+* Check the version of the collector you have installed by running the following command: `pip show sumologic-netskope-collector`
+* If you are not running the latest collector, do the following:
+1. Disable your cron job.
+2. Stop all existing collector processes.
+3. Run the following command to upgrade your collector: `pip install sumologic-netskope-collector --upgrade`
+4. Enable the cron job.
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/netskope-legacy-collection.md:118]:
```suggestion
   1. Disable your cron job.
   2. Stop all existing collector processes.
   3. Run the following command to upgrade your collector: `pip install sumologic-netskope-collector --upgrade`
   4. Enable the cron job.
```

File: docs/integrations/security-threat-detection/netskope.md [hunk 10]
```diff
@@ -23,7 +23,7 @@ For more information on Netskope, refer to the Netskope [documentation](https://
 
 ## Collect Logs for Netskope
 
-To collect logs from the Netskope platform, use the [new Cloud to Cloud Integration for Netskope](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/Netskope-Source) to create the source and use the same source category while installing the app.
+To collect logs from the Netskope platform, use the [new Cloud to Cloud Integration for Netskope](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/Netskope-Source) to create the source and use the same source category while installing the app. For the legacy collection, see [Old Collection method](/docs/integrations/security-threat-detection/netskope-legacy-collection/).
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/netskope.md:26]:
```suggestion
To collect logs from the Netskope platform, use the [new Cloud to Cloud Integration for Netskope](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/Netskope-Source) to create the source and use the same source category while installing the app. You can view the Netskope legacy collection method [here](/docs/integrations/security-threat-detection/netskope-legacy-collection/).
```

**Comment by JainM6** [comment]
> Looks like an extra line in the steps. Please check.

@stacykor Thank you for pointing this out. It's fixed.

**Pull request merged**